protobuf/protobuf.go

```diff
@@ -0,0 +1,75 @@
+package protobuf
+
+import (
+	"fmt"
+
+	"github.com/Masterminds/semver"
+	"github.com/cloudchacho/hedwig-go"
+	hedwigProtobuf "github.com/cloudchacho/hedwig-go/protobuf"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type FirehoseEncoderDecoder struct {
+	*hedwigProtobuf.EncoderDecoder
+	typeUrls map[hedwig.MessageTypeMajorVersion]string
+}
+
+func (fcd *FirehoseEncoderDecoder) DecodeData(messageType string, version *semver.Version, data interface{}) (interface{}, error) {
+	return data, nil
+}
+
+// EncodeData encodes the message with appropriate format for transport over the wire
```

Comment:
comment is invalid. encode data encodes in format appropriate for _storage_ to firehose files.